samples/rust/hello-foundry-local/src/main.rs

```diff
@@ -0,0 +1,62 @@
+use anyhow::Result;
+use foundry_local::FoundryLocalManager;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // Set up logging
+    env_logger::init_from_env(
+        env_logger::Env::default().filter_or(env_logger::DEFAULT_FILTER_ENV, "info"),
+    );
+
+    println!("Hello Foundry Local!");
+    println!("===================");
+
+    // Create a FoundryLocalManager instance with default options
+    // This will start the Foundry service if not running
+    println!("\nInitializing Foundry Local manager...");
+    let mut manager = FoundryLocalManager::new(None, None, None).await?;
```

Comment:
This is a good example where the builder pattern is preferred to avoid `(None, None, None)` which is not very idiomatic. For examples, see how a [Windows service is constructed here](https://crates.io/crates/windows-services) or how a [registry key is constructed here](https://crates.io/crates/windows-registry).

Reply:
Updated to use builder, since the common case will probably be emptier construction.